custom_components/easee/sensor.py

```diff
@@ -61,21 +59,21 @@ def watts_to_kilowatts(value):
     "total_power": {
         "key": "state.totalPower",
         "attrs": [],
-        "units": "W",
-        "convert_units_func": watts_to_kilowatts,
+        "units": "kW",
```

Comment:
When I tested I got W for this. Are you sure you got kW?

Reply:
Quite sure, but I'll double check.

Reply:
Here's what I get when running from CLI where I've only rounded the numbers:

![image](https://user-images.githubusercontent.com/54450177/87942408-d24a1200-ca9c-11ea-8d8c-cce7cf01bc62.png)

Reply:
Let's merge it and I can test again